.github/workflows/test.yml

```
@@ -56,6 +56,10 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Compose
+        uses: ndeloof/install-compose-action@v0.0.1
```

Use SHA, its more secure:

```suggestion
        uses: ndeloof/install-compose-action@c036dce35dc2a23597c03aa05f32347b2cfa0437
```